src/content/scripting/script.cc

```diff
@@ -803,7 +803,7 @@ std::string Script::convertToCharset(const std::string& str, charset_convert_t c
         return _f2i->convert(str);
     case J2I:
         return _j2i->convert(str);
-    default:
+    case I2I:
```

Comment:
Doesnt this change the behaviour of the method?

Reply:
No. Initially I moved the default section out of the switch but then got an unhandled switch statement warning. Note how the return how matches the case name.